cloud-sql/mysql/servlet/src/main/java/com/example/cloudsql/ConnectorConnectionPoolFactory.java

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example.cloudsql;
+
+import com.zaxxer.hikari.HikariConfig;
+import com.zaxxer.hikari.HikariDataSource;
+import edu.umd.cs.findbugs.annotations.SuppressFBWarnings;
+import javax.sql.DataSource;
+
+
+public class ConnectorConnectionPoolFactory extends ConnectionPoolFactory {
+
+  // Saving credentials in environment variables is convenient, but not secure - consider a more
+  // secure solution such as https://cloud.google.com/kms/ to help keep secrets safe.
+  private static final String INSTANCE_CONNECTION_NAME =
+      System.getenv("INSTANCE_CONNECTION_NAME");
+  private static final String INSTANCE_UNIX_SOCKET = System.getenv("INSTANCE_UNIX_SOCKET");
+  private static final String DB_USER = System.getenv("DB_USER");
+  private static final String DB_PASS = System.getenv("DB_PASS");
+  private static final String DB_NAME = System.getenv("DB_NAME");
+
+  @SuppressFBWarnings(
+      value = "USBR_UNNECESSARY_STORE_BEFORE_RETURN",
+      justification = "Necessary for sample region tag.")
+  public static DataSource createConnectionPool() {
+    // [START cloud_sql_mysql_servlet_connect_connector]
```

Comment:
This should probably move up to include the package, or at least the imports?

Reply:
moved the region tags to include imports